back-end/controller.js

```diff
@@ -170,4 +158,25 @@ const getVisitorsByName = async (request, response) => {
 
 }
 
-module.exports = { addNewVisitor, getAdminAuthorization, getVisitorsBySignIn, signOutOneVisitorById, signOutAllVisitors, getVisitorsByName }
+const destroyAdminAuthorization = async (request, response) => {
+    request.session.destroy()
+    store.all((error, sessions) => { sessions.forEach((session) => console.dir(session)) })
+    response.status(200).send()
+}
+
+const optionControl = async (request, response) => {
+    response.status(200).send()
+}
+
+const clearSessionStore = async (request, response) => {
+    store.clear((error) => {
+        console.log("Session store cleared")
```

Comment:
maybe we don't need the console logs now

Reply:
I remember now the console logs will only show in the terminal in the backend, so maybe it's ok to keep them in, therefore, will merge